client/src/components/PageHeader/PageHeader.js

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { FiMenu } from "react-icons/fi";
+import { MdClose } from "react-icons/md";
+import styles from "./PageHeader.module.css";
+import { PropTypes } from "prop-types";
+
+const MenuDropdown = ({ onClose }) => {
+  const navigate = useNavigate();
+  const handleLogOut = () => {
+    // TODO : API call for logout
+    navigate("/");
+  };
+
+  return (
+    <div className={styles.navDropdown}>
+      <NavButton hamburger={false} onClick={onClose} />
+      <div className={styles.menuItemContainer}>
+        <MenuDropdownItem
+          label={"Calendar"}
+          action={() => navigate("/calendar")}
+        />
+        <MenuDropdownItem
+          label={"Task History"}
+          action={() => navigate("/task_history")}
+        />
+        <MenuDropdownItem
+          label={"Insights"}
+          action={() => navigate("/insights")}
+        />
+        <MenuDropdownItem label={"Log out"} action={handleLogOut} />
+      </div>
+    </div>
+  );
+};
+
+const MenuDropdownItem = ({ label, action }) => {
+  return (
+    <button className={styles.navDropdownItem} onClick={action}>
+      {label}
+    </button>
+  );
+};
+
+const NavButton = ({ hamburger, onClick }) => {
```

Comment:
I think this is better to be placed in the same component as the Sidebar since these 2 parts should go hand in hand with each other

Reply:
`NavButton` is used in both the header and the modal (see `Sidebar` and `styles.navDropdown`). The implementation is similar to the modal/pop-up (e.g., `TaskDetails`). I'm not sure how to place it only in the Sidebar

Reply:
We can disregard this comment. I interpreted the NavButton to be something else.

Reply:
Something that confused me was just the use of the same button for closing and opening the menu. In other instances of CSS frameworks, they will actually be different things rather than the same component. For our implementation this is okay though